run_dir/static/js/flowcell.js

```diff
@@ -1,25 +1,25 @@
 /* flowcell.html */
 
-function display_undetermined(lane){
```

Comment:
I think this is a bit of a downgrade. From what I read, an alternative could also be 
```
const display_undetermined = function(lane) {
```
would that work/be better?

Reply:
I think I had that code to begin with, right? I don't know, did you change it, Anand?